src/main/user-impl/java/com/mysql/cj/jdbc/ha/plugins/AWSSecretsManagerPlugin.java

```diff
@@ -240,6 +242,12 @@ private boolean updateSecret(boolean forceReFetch) throws SQLException {
     return fetched;
   }
 
+  private void attemptToUpdateCurrentHostInfoWithSecrets(HostInfo hostInfo) {
```

Comment:
super nit: I think just `updateCurrentHostInfoWithSecrets` will work for this method name

Reply:
Done!